notebooks/02_model/surprise_svd_deep_dive.ipynb

```diff
@@ -748,20 +732,25 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## References\n",
+    "## Additional Reading\n",
     "\n",
-    "1. Ruslan Salakhutdinov and Andriy Mnih. Probabilistic matrix factorization. 2008. URL: http://papers.nips.cc/paper/3208-probabilistic-matrix-factorization.pdf\n",
```

Comment:
We may want to keep these since they are the papers cited in the Surprise docs. In fact (1) is closest to the SVD formulation.

Reply:
Yes thanks for the reference.  Will add it back once I have a little more time.